datadriven/tests/tbin/tlearner/test_Classifier.py

```diff
@@ -135,7 +135,7 @@ def testLearnDataWithFolding(self,):
         dim = 6
         l = 0.00001
         self.classifier = Classifier()
-        dataContainer = ARFFAdapter(pathlocal + "/../../../datasets/liver/liver-disorders_normalized.arff").loadData()
+        dataContainer = ARFFAdapter(pathlocal + "/../../../datasets/liver/liver-disorders_normalized_small.arff").loadData()
```

Comment:
Changing from the normal to small dataset makes the test fail. Was this intentional?

Reply:
Just fixed this together with @spc90